### ISSUE TITLE
stdin: is not a tty Error

### ISSUE THREAD

**Issue by pramttl** [osuosl/vagrant-ganeti#1]
I frequently get this "stdin: is not a tty Error" when I run "vagrant up" which does not let the command complete (stops), until I run "vagrant up" again from another terminal instance.
Thinking on a fix for this. Perhaps changes to the postinstall.sh script?
Here is the complete trace I get:

Bringing machine 'node1' up with 'virtualbox' provider...
[node1] Importing base box 'ganeti-ubuntu-12.04-20130411'...
[node1] Matching MAC address for NAT networking...
[node1] Setting the name of the VM...
[node1] Clearing any previously set forwarded ports...
[node1] Creating shared folders metadata...
[node1] Clearing any previously set network interfaces...
[node1] Preparing network interfaces based on configuration...
[node1] Forwarding ports...
[node1] -- 22 => 2222 (adapter 1)
[node1] Running any VM customizations...
[node1] Booting VM...
[node1] Waiting for VM to boot. This can take a few minutes.
[node1] VM booted and ready for use!
[node1] Setting hostname...
[node1] Configuring and enabling network interfaces...
[node1] Mounting shared folders...
[node1] -- /vagrant
[node1] -- /tmp/vagrant-puppet/manifests
[node1] -- /tmp/vagrant-puppet/modules-0
[node1] Running provisioner: VagrantPlugins::Puppet::Provisioner::Puppet...
Running Puppet with node1.pp...
stdin: is not a tty

**Comment by ramereth** [comment]
Yes it looks like I need to fix my postinstall.sh and add the fix from [upstream ](https://github.com/patcon/veewee/commit/efeff87fb14d98efe773078c22775af41cde2f9e). In the meantime you should be able to run `vagrant provision node1` after you see that error and it should work then (at least according to the upstream bug report). I will update the images next week sometime to fix that. Please let me know if that works.
